include/nccl_ofi_param.h

```diff
@@ -92,8 +92,15 @@ OFI_NCCL_PARAM_STR(exclude_tcp_if, "EXCLUDE_TCP_IF", "lo,docker0");
  */
 OFI_NCCL_PARAM_INT(gdr_flush_disable, "GDR_FLUSH_DISABLE", 0);
 
+/*
```

Comment:
We also need a change in README documentation to describe the new configuration.